src/main/java/io/georgeous/mcgenerations/commands/player/banish/BanishCommand.java

```
@@ -31,19 +31,28 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
             return true;
         }
 
-        if (args.length < 2 || args.length > 3) {
-            Notification.errorMsg(player, "Usage: /banish <Name> <Reason>");
-            return true;
-        }
-
         RolePlayer senderRole = RoleManager.get().get(player);
         if(senderRole == null){
             Notification.errorMsg(player, "You dont have a role");
             return true;
         }
 
+        if (args.length < 2 || args.length > 3) {
+            Notification.errorMsg(player, "Usage: /banish <Name> <Reason>");
+            return true;
+        }
+
         if(args[0].equalsIgnoreCase("request")){
-            handleRequestResponse(UUID.fromString(args[2]), args[1], senderRole);
+            if(args.length != 3){
+                return true;
+            }
+            UUID uuid;
+            try {
+                uuid = UUID.fromString(args[2]);
+            } catch(IllegalArgumentException exception){
+                return true;
```

If the UUID cannot be parsed this will fail silently. Are you sure you want this? Might be better to return false, send a notification and/or at least log to console.